src/flash.rs

```diff
@@ -0,0 +1,313 @@
+//! IS25LP064A: 64Mbit/8Mbyte flash memory
+//!
+//! https://www.issi.com/WW/pdf/25LP032-64A-B.pdf
+//!
+
+// 133Mhz
+// bank 1
+//  pin_group[DSY_QSPI_PIN_IO0] = dsy_pin(DSY_GPIOF, 8);
```

Comment:
Dead code?

Reply:
it is the c++ source that I used to get the info for how the pins were mapped.. i've ended up finding that much easier to grok than the other info around.. I'll ditch it though